dali/test/python/test_optical_flow.py

```diff
@@ -12,7 +12,7 @@
 batch_size = 1
 sequence_length = 2
 dali_extra_path = os.environ['DALI_EXTRA_PATH']
-image_dir= os.path.join(dali_extra_path, "/db/optical_flow/slow_preset/two_frames")
+image_dir = dali_extra_path + "/db/optical_flow/slow_preset/two_frames/"
```

Comment:
Why?

Reply:
IMHO `os.path.join()` is evil.
This:
`image_dir= os.path.join(dali_extra_path, "/db/optical_flow/slow_preset/two_frames")` is resolved to `/db/optical_flow/slow_preset/two_frames` cause of the `/` at the beginning.

On top of that, it doesn't make any sense using `os.path.join()` with hardcoded strings

Reply:
So you can use:
```
os.path.join(dali_extra_path, "db", "optical_flow", "slow_preset", "two_frames")
```
I would prefer portable solution than hardcodes, especially with zero effort as in this case.

Reply:
done

Reply:
Wouldn't that work:
```suggestion
image_dir = os.path.join(dali_extra_path, "db/optical_flow/slow_preset/two_frames/")
```
?
In any case, I think 
`os.path.join(dali_extra_path, "db", "optical_flow", "slow_preset", "two_frames")`
is worse than 
`dali_extra_path + "/db/optical_flow/slow_preset/two_frames/"` - it's less readable, and "//" is still accepted as "/".
Unless you want to run it on windows, which won't work at all in any foreseeable future.